app/src/main/java/com/github/polybooks/utils/DebugUtil.kt

```diff
@@ -0,0 +1,35 @@
+package com.github.polybooks.utils
+
+import com.github.polybooks.core.Course
+import com.github.polybooks.core.Field
+import com.github.polybooks.core.Semester
+
+val mockFields : List<Field> = listOf(
```

Comment:
All these could go directly in the DummyInterestDatabase instead of the Utils package

Reply:
ok yes you are probably right

Reply:
done in PR #114